Practice_9/Calculator.cs

```diff
@@ -4,5 +4,40 @@ namespace Practice_9
 {
     public class Calculator
     {
+        public double x;
+        public double y;
```

Comment:
why do yopu need these x and y variables + constructor for them? the Calculatir class doesn't need to have such parameters are they anyway are passed as methods parameters.